

testBlock = {
    def 1
}

int = make_stack(testBlock)
int.run

assert(int.error_message == "Expected identifier after def, found: 1")
